front/app/components/IdeaButton/index.tsx

```diff
@@ -323,16 +323,22 @@ const IdeaButton = memo<Props & InjectedIntlProps>(
                   disabled={!enabled}
                   ariaDisabled={false}
                 >
-                  <FormattedMessage
-                    {...getInputTermMessage(inputTerm, {
-                      idea: messages.submitYourIdea,
-                      option: messages.addAnOption,
-                      project: messages.addAProject,
-                      question: messages.addAQuestion,
-                      issue: messages.submitAnIssue,
-                      contribution: messages.addAContribution,
-                    })}
-                  />
+                  {project.attributes.participation_method === 'ideation' && (
```

Comment:
Suggestion: non blocking:

In addition to the discussion on the if conditions, writing it like this makes it look like both could be rendered at the same time. We could explicitly state intent by always rendering the `FormattedMessage` without any condition. The only condition if needed here is the one on the props that we can extract out above into a variable.

```
<FormattedMessage {...translatedMessage} />
```